ci/insert_libtelio_version.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python3
+
+import argparse
+import os
+import replace_string
+import sys
+from build_libtelio import LIBTELIO_CONFIG, PROJECT_CONFIG, PROJECT_ROOT
+from rust_build_utils.rust_utils_config import GLOBAL_CONFIG
+
+VERSION_PLACEHOLDER = "VERSION_PLACEHOLDER@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@"
+
+
+def insert_version_to_libtelio_binaries_in_dir(new_version: str, path: str):
```

Comment:
Quick thought, would you be able to ensure, that in case there is no replacement (e.g. zero placeholders found in file) -> this function throws an error, in the hopes of ensuring, that we do not miss somehow version replacement.

Reply:
`replace_string_in_file` throws an exception if original string, that we want to replace, is not found